### PR TITLE
Bump package lock

### LINKED ISSUE #600: Bump package-lock to have the latest package dependencies
**Is your feature request related to a problem? Please describe.**
Currently the package-lock does not have the latest dependencies (and it differs via the npm version the local user is running to produce the package-lock)

**Describe the solution you'd like**
Update the package-lock

**Additional context**
There seems to be a Travis error with the latest dependencies pulled into the application.

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #600 

_This PR removes all local `/node_modules` and the `package-lock.json` and does a fresh `npm install` with node@8.11.4 and npm@6.4.1_

- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/lib/globalStyles.test.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ describe('globalStyles', () => {
 
     await import('./globalStyles');
 
-    const injectGlobalArgs = styledComponents.injectGlobal.mock.calls[0];
+    const injectGlobalArgs = styledComponents['injectGlobal'].mock.calls[0]; // eslint-disable-line dot-notation
```
Comment (pjlee11) [inline src/app/lib/globalStyles.test.js]: For some reason (that I've not been able to prove) Travis will fail on the commit of the new package-lock.json if this line uses dot notation to access the `injectGlobal` key of the `styleComponents` object. 
The commit `c0790206a46fc635b0f9817a7d85f16208fa99bb` in this branch shows this as Travis fails as per the following screenshot:

image

I have a few hunches as to why Travis is failing: 
1) A dependency has changed with a fresh install which is causes a parsing error of `globalStyles.test.js` somewhere in the application prior to test running. However I would counter this saying that on my local machine I'm able to replicate the Travis error by making any line change to `globalStyles.test.js` without the package-lock change. 
2) Somewhere in the minification process of the application there is an error being caused which is not being surfaced to the failing test. I would counter this by saying on different local machine this test seems to pass or fail regardless of new line additions.
3) The test is passing for Travis on latest and for my local with the package-lock changes due to the minification process seeing no change to the file and uses a cached version of the file. This could explain the issue as locally the test would pass or fail with line changes varying on the users local node_modules cache. With node_modules being installed fresh on Travis we therefore would see this issue on ci and not on certain local machine. It would also explain how locally with node_modules removed and fresh install how any change to the test file would update the minification cache and reveal the same error as seen on Travis

**Comment by sareh** [comment]
@pjlee11 Could you please also update the npm version in the `.npmrc` file?

File: src/app/lib/globalStyles.test.js [hunk 2]
```diff
@@ -6,7 +6,8 @@ describe('globalStyles', () => {
 
     await import('./globalStyles');
 
-    const injectGlobalArgs = styledComponents.injectGlobal.mock.calls[0];
+    // For explaination on the dot nation ignore see https://github.com/bbc/simorgh/pull/601/files#r214874027
+    const injectGlobalArgs = styledComponents['injectGlobal'].mock.calls[0]; // eslint-disable-line dot-notation
```
Comment (radiocontrolled) [inline src/app/lib/globalStyles.test.js:10]: I could not find the link to the explanation. What is the relationship between eslint-disabling this line and fixing the travis error?

File: package-lock.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@babel/code-frame": {
       "version": "7.0.0-beta.44",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0-beta.44.tgz",
+      "resolved": "http://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0-beta.44.tgz",
```
Comment (pjlee11) [inline package-lock.json:9]: I'm not sure why this is changing to `http` I initially thought it was due to my npm config however the registry is `https` so I'm surprised by this lock output

**Comment by pjlee11** [comment]
TL;DR - I think this was a black magic parsing error due to the package-lock being bundled by a different npm version to the npm version on ci running sed package-lock.

Explanation for the dot notation change. 

1) it could be a dependency change issue. but this wouldn't make sense locally when I make a line change to the file on latest and the test fails.
2) it could be that the file minification is being cached locally but not on Travis.
3) it could be a parsing error hence changing from dot notation seems to work.
4) it could be a version issue between the node or npm causing the package-lock to have knock on affect to the parsing of the code.

**Comment by pjlee11** [comment]
Closing in favour of #602
